modules/mw.cx.TargetArticle.js

```
@@ -8,12 +8,16 @@
  * @param {Object} config Translation configuration
  * @param {mw.cx.SiteMapper} config.siteMapper SiteMapper instance
+ * @param {string} config.campaign Campaign name for targeting purposes
  */
 mw.cx.TargetArticle = function MWCXTargetArticle(translation, veTarget, config) {
 	this.translation = translation;
 	this.veTarget = veTarget;
 	this.config = config;
 	this.siteMapper = config.siteMapper;
+	this.campaign = config.campaign || '';;
+
+	console.log('MWCXTargetArticle: campaign: ' + this.campaign);
```

ESLint

[error] Unexpected console statement.

(no-console)